src/biogeophys/CanopyStateType.F90

```diff
@@ -243,6 +246,14 @@ subroutine InitHistory(this, bounds)
        call hist_addfld2d (fname='VEGWP',  units='mm', type2d='nvegwcs', &
             avgflag='A', long_name='vegetation water matric potential for sun/sha canopy,xyl,root segments', &
             ptr_patch=this%vegwp_patch)
+       this%vegwp_ln_patch(begp:endp,:) = spval
+       call hist_addfld2d (fname='VEGWPLN',  units='mm', type2d='nvegwcs', &
```

Comment:
- [ ] Make these default active (and QHR and VDP as well)

Reply:
making these defaults active did not resolve the issue.